### ISSUE TITLE
resolve substrate v3 pallet-did dependency

### ISSUE THREAD

**Issue by vgalleg4** [IoTone/substrate-enterprise-sample-v3#10]
Per @Damfortx8 it appears we have a dependency on pallet-did, however we need the porting done to v3.  See https://github.com/substrate-developer-hub/pallet-did/pull/15

For unknown reason the pull request that brings in v3 support is not accepted.  Maybe it does not matter if we use the branch directly.

**Comment by truedat101** [comment]
What is this one for?

**Comment by vgalleg4** [comment]
@truedat101  DID is a pallet

**Comment by truedat101** [comment]
I've requested some feedback on the pull request review (why it's waiting or blocked).

**Comment by truedat101** [comment]
Rather than get blocked on this, I think we can just copy the pull request code branch directly into our codebase.

**Comment by Damfortx8** [comment]
This is already done, i updated the pallet and is in the project  [](https://github.com/IoTone/substrate-enterprise-sample-v3/tree/feature/did)

**Comment by truedat101** [comment]
Looks like we took the approach recommended here https://github.com/substrate-developer-hub/pallet-did/pull/15 and include the code directly in our repo.
